src/format-cleanup-changes.ts

```diff
@@ -0,0 +1,27 @@
+export interface CleanupChange {
+  prNumber: number;
+  prTitle: string;
+  prURL: string;
+}
+
+export type CleanupChangesByFile = Map<string, CleanupChange[]>;
+
+export function formatCleanupChanges(changes: CleanupChange[]): string {
+  if (!changes.length) return '';
+
+  const prMap = new Map<number, CleanupChange>();
```

Comment:
this is for deduping, right? (maybe add a comment since it took me a sec to realize)